src/bundle/Resources/public/scss/_content-edit.scss

```diff
@@ -70,3 +70,9 @@
         padding: 0 calculateRem(20px);
     }
 }
+
+.ez-content-item__errors-wrapper {
+    font-style: italic;
+    color: $ez-color-danger;
+    font-size: calculateRem(12px);
```

Comment:
Due to consistency with the rest of validation messages displayed, Could you please change color to `$ez-color-warning-dark` and font-size to `16px`. Thanks!